### DISCUSSION TITLE
Idea: Fun Alternative for Onboarding New Users

### DISCUSSION THREAD

**Discussion by nickjlb** [mastodon/mastodon#24803] [General]
At the moment, there seems to be a lot of controversy over the corralling of users toward mastodon.social. I see the benefits: An easier more seamless introduction to the fediverse, but on the other hand, it seems to go against the nature of decentralization and people might not find communities they otherwise would, and so they leave.

Rather than having design that focuses on new users toward that instance, instead (and hear me out), gamifying the process. Dedicating time to making a sort of quiz/questionaire (in the Buzzfeed/personality quiz style) that after going through X number of questions, gives users a small list of servers based on their interests or what they seem to be looking for. Instead of having the single button to join, new users are taken through this process to help them narrow down their focus while still avoiding the much-discussed overwhelm of searching through all the possible servers. Instead of the process being a drag or tedious, it both makes the process fun, avoids centralization, and gets users much closer to servers that they will actually participate in.

I'm not blind to some of the issues: What questions would be asked, how do we organize servers, inherent biases towards certain instances over others, what instances make the cut and which don't? But, I think these issues are still preferable to having a single option. LMK what y'all think.

**Comment by Andrewblasco** [comment]
I think this could be useful and funny. But maybe as an option in the sign up process. Like:

- Option a) Predefined server 
- Option b) Mastodon test "the server chooses the user" (I hope you get the joke). It could be presented with the little elephant of Mastodon characterized with different features to represent different server focus (photography, general, activism, regional...).
- Option c) Dive into all the servers and choose one!

Many people maybe don't want to answer questions or they don't know what implies certain questions because they don't use Mastodon already (for example, a question about CW policy in the server).

**Reply by shaedrich** [reply]
If people want a less gamified version of that, they could be provided with an interface to just "follow" hashtags (which the later set up account would then already follow) and then provided with an instance, where these hashtags are either featured or have high activity
